data-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/data/rest/RestContractTest.java

```diff
@@ -16,121 +16,121 @@
-
-    private AvroSerialiser<Employee> avroSerialiser;
+    private AuthorisedRequestsRepository requestsRepository;
+    @Autowired
+    private TestRestTemplate restTemplate;
+    @Autowired
+    private MockMvc mockMvc;
 
     @BeforeEach
-    void setUp() throws JsonProcessingException {
-        AuditServiceMock.stubRule(auditMock, MAPPER);
-        auditMock.start();
-
-        PalisadeServiceMock.stubRule(palisadeMock, MAPPER, RESOURCE);
-        palisadeMock.start();
-
-        MAPPER.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-        avroSerialiser = new AvroSerialiser<>(Employee.class);
-    }
-
-    @AfterEach
-    void tearDown() {
-        auditMock.stop();
-        auditMock.resetAll();
-
-        palisadeMock.stop();
-        palisadeMock.resetAll();
+    void setUp() {
+        // Given - AvroSerialiser added to data-service
+        dataService.addSerialiser(DataFlavour.of(Employee.class.getTypeName(), "avro"), AVRO_SERIALISER);
+
+        // Given - Request for "token" and CURRENT_PATH is authorised
+        requestsRepository.save(new AuthorisedRequestEntity(
+                "token",
+                new User().userId("test-user"),
+                new FileResource().id(CURRENT_PATH)
+                        .serialisedFormat("avro")
+                .type(Employee.class.getTypeName())
+                .connectionDetail(new SimpleConnectionDetail().serviceName("data-service"))
+                .parent(new SystemResource().id("/")),
+                new Context(),
+                new Rules<>()
+        ));
     }
 
     @Test
     void testContextLoads() {
-        assertThat(serviceMap).isNotNull();
-        assertThat(serviceMap).isNotEmpty();
+        assertThat(dataService).isNotNull();
+        assertThat(restTemplate).isNotNull();
+        assertThat(mockMvc).isNotNull();
     }
 
     @Test
     void testIsUp() {
-        Response health = client.getHealth();
-        assertThat(health.status()).isEqualTo(HttpStatus.OK.value());
+        ResponseEntity<JsonNode> health = restTemplate.getForEntity("/actuator/health", JsonNode.class);
+        assertThat(health.getStatusCode()).isEqualTo(HttpStatus.OK);
     }
 
     @Test
-    void testReadChunked() throws IOException {
-        // Given - ReadRequest created
-        ReadRequest readRequest = new ReadRequest().token("token").resource(RESOURCE);
-        readRequest.setOriginalRequestId(new RequestId().id("original"));
-
-        // Given - AvroSerialiser added to Data-service
-        client.addSerialiser(DataFlavour.of(Employee.class.getTypeName(), "avro"), avroSerialiser);
-
+    void testReadChunked() throws Exception {
         // Given - the file contains the expected data
-         avroSerialiser.serialise(Stream.of(new Employee()), new FileOutputStream("src/contract-tests/resources/data/employee_file0.avro"));
+        // AVRO_SERIALISER.serialise(Stream.of(new Employee()), new FileOutputStream(CURRENT_PATH));
```

Comment:
Can we delete the commented line of code or is it required?

Reply:
Commented that it's not needed, just how the data file was produced